cifar10/cifar10_train.py

```diff
@@ -251,7 +249,33 @@ def train_model(
 
     trainer.fit(model, dm)
     trainer.test()
-    torch.save(model.state_dict(), os.path.join(model_save_path, 'resnet.pth'))
+    torch.save(model.state_dict(), os.path.join(model_save_path, "resnet.pth"))
+
+    generate_confusion_matrix(
```

Comment:
I will move the confusion matrix generation to Visualisation component, once the KFP volumes related logics are ready.